week-5/Homework/mandatory/3-project/js/main.js

```diff
@@ -93,11 +93,8 @@ let email= document.getElementById('exampleInputEmail1');
         email.style.backgroundColor = "red";
         alert("Email is not correct");
       }
-
-       
-        
-        
-        else {
+          
+          else {
             alert ("Thank you for filling out this form")
           
         }
```

Comment:
The only amendment to exercise 2 would be to remove the bullet point from each <li> tag - ul {list-style: none}
Exercise 3 is correct.